core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```diff
@@ -455,12 +454,29 @@ fi
 # --- Install required packages unless present ---
 
 # repos can be enabled or disabled using the yum-config-manager command, which is provided by yum-utils package
-for package in 'yum-utils' 'wget' 'curl'; do
+for package in 'yum-utils' 'wget' 'curl' 'tar'; do
 	if ! is_package_installed "$package"; then
 		install_package "$package"
 	fi
 done
 
+# --- Download and setup Crane for downloading images ---
+
+if [[ -z "${CRANE}" ]] || [ $(wc -l <<< "${CRANE}") -ne 1 ] ; then
+    exit_with_error "Crane binary download path undefined or more then one download path defined"
+else
+    file=$(head -n 1 <<< "${CRANE}")
+    echol "Downloading crane from: ${file}"
+    download_file "${file}" "${SCRIPT_DIR}"
+    tar_path="${SCRIPT_DIR}/${file##*/}"
+    echol "Unpacking crane from ${tar_path} to ${CRANE_BIN}"
+    tar -xf "${tar_path}" --directory ${SCRIPT_DIR} "crane" --overwrite
+    chmod +x "${CRANE_BIN}"
```

Comment:
```suggestion
    run_cmd tar -xzf "${tar_path}" --directory "${SCRIPT_DIR}" "crane" --overwrite
    [[ -x "${CRANE_BIN}" ]] || run_cmd chmod +x "${CRANE_BIN}"
```

tar has "auto-detect" feature to recognize compressed archive but I'm not sure about older versions so it could be better to use 'z' option to be on the safe side 😉